fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/expressions/Comparisons.java

```diff
@@ -825,6 +837,10 @@ public String toString() {
         private final Type type;
         @Nonnull
         protected final String parameter;
+        @Nullable
+        protected Object comparand;
```

Comment:
I don't think we want to have a `comparand` member on this class. I don't think it's used, even in the new code?